storage/src/main/java/tech/pegasys/teku/storage/server/leveldb/LevelDbTransaction.java

```diff
@@ -92,7 +93,10 @@ public void commit() {
     applyUpdate(
         () -> {
           try {
+            long startTime = System.nanoTime();
             db.write(writeBatch);
+            long endTime = System.nanoTime();
+            DB_LOGGER.onDbOpAlertThreshold("LevelDB update", startTime, endTime);
```

Comment:
```suggestion
            DB_LOGGER.onDbOpAlertThreshold("Transaction commit", startTime, endTime);
```

Just to make it a little more user centric.

Reply:
I also renamed the `KvStoreDatabase update` to `Block Import`